tests/nopytest_convertReferenceCatalog.py

```diff
@@ -250,6 +246,12 @@ def test_getCatalog(self):
         self.assertFloatsEqual(newcat[:len(catalog)]['coord_ra'], catalog['coord_ra'])
         self.assertFloatsEqual(newcat[:len(catalog)]['coord_dec'], catalog['coord_dec'])
 
+    def test_setCoordinateCovariance(self):
+        catalog = self._createFakeCatalog(nOld=10, nNew=0)
+
+        with self.assertRaises(NotImplementedError):
+            self.worker._setCoordinateCovariance(catalog[0], self.fakeInput[0])
```

Comment:
I'm not sure this test accomplishes much: we've defined it as not implemented in the base class.

Reply:
I'm pretty sure this test was requested in earlier comments.